CHANGES.rst

```diff
@@ -1,6 +1,11 @@
 0.18.4 (unreleased)
 ===================
 
+cube_build
+----------
+
+- Added support for cross-dichroic configurations
```

Comment:
Add the PR number to the end of the line, e.g. [#5722]

Reply:
Fixed